### ISSUE TITLE
toolbar test failing randomly

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-ngx#2808]
```
Chrome Headless 62.0.3202.94 (Linux x86_64) ToolbarComponent should create FAILED
	Error: Expected 22 to equal 0.
	    at <Jasmine>
	    at http://localhost:9876/_karma_webpack_/src/lib/toolbar/toolbar.component.spec.ts:102:57
	    at step (http://localhost:9876/home/travis/build/SAP/fundamental-ngx/node_modules/tslib/tslib.es6.js:100:1)
	    at Object.next (http://localhost:9876/home/travis/build/SAP/fundamental-ngx/node_modules/tslib/tslib.es6.js:81:45)
```

```
Chrome Headless 62.0.3202.94 (Linux x86_64) ERROR
  An error was thrown in afterAll
  Error: Expected 27 to equal 0.
      at <Jasmine>
      at http://localhost:9876/_karma_webpack_/src/lib/toolbar/toolbar.component.spec.ts:102:57
      at step (http://localhost:9876/home/travis/build/SAP/fundamental-ngx/node_modules/tslib/tslib.es6.js:100:1)
      at Object.next (http://localhost:9876/home/travis/build/SAP/fundamental-ngx/node_modules/tslib/tslib.es6.js:81:45)
```

**Comment by stefanoScalzo** [comment]
There are actually 44 failed tests, it might be a larger issue and we should bring this up.

**Comment by droshev** [comment]
@stefanoScalzo did we fix that?

**Comment by droshev** [comment]
fixed with #3019
